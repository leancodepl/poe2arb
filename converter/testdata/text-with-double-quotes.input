[
    {
        "term": "textWithDoubleQuotes",
        "definition": "Those are some \"quotes\".",
        "context": "",
        "term_plural": "",
        "reference": "",
        "comment": ""
    }
]