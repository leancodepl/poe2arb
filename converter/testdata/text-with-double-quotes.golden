{
    "@@locale": "en",
    "textWithDoubleQuotes": "Those are some \"quotes\".",
    "@textWithDoubleQuotes": {}
}
